platform/ios/src/MGLMapView.mm

```diff
@@ -4690,13 +4695,15 @@ - (void)applyIconIdentifier:(NSString *)iconIdentifier toAnnotationsWithImageReu
 
 #pragma mark - User Location -
 
-- (void)setLocationManager:(id<MGLLocationManager>)locationManager
+- (void)setLocationManager:(nullable id<MGLLocationManager>)locationManager
```

Comment:
Since the property is `null_resettable` does this need to be? (I don't know the Obj-C rules here.)

Reply:
The property is `nullable` and since I'm overriding the setter it should be in accordance.